Makefile

```diff
@@ -14,6 +14,8 @@ DIR = $(shell pwd)
 
 ANT ?= ant
 ANT_OPTIONS =
+KBASE_COMMON_JAR = kbase/common/kbase-common-0.0.20.jar
```

Comment:
The purpose is to support JENKINS tests (which we stopped to run unfortunately) where you compile temporary common jar from develop branch of java_common.